src/_data/team/pez-cuckow.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Pez Cuckow",
+    "title": "Senior Software Engineer",
+    "twitter": "Pezmc",
+    "github": "Pezmc",
+    "blog": "https://www.pezcuckow.com",
+    "country": "BE"
```

Comment:
Nationality, or current country? Chosen BE here which gives the page more flags.

Reply:
Current country for the timezone etc.